daemon/main.go

```diff
@@ -106,6 +107,8 @@ func init() {
 	flags.BoolVar(&config.KeepConfig, "keep-config", false,
 		"When restoring state, keeps containers' configuration in place")
 	flags.StringVar(&labelPrefixFile, "label-prefix-file", "", "File with valid label prefixes")
+	flags.StringSliceVar(&validLabels, "labels", []string{},
```

Comment:
Why do we need it for the client? I would rather not complicate this unless required. Restarting the daemon is possible without losing state so unless we get a direct request for making this changeable at runtime I don't see a need for it.